filebeat/module/kibana/log/test/test.log-expected.json

```diff
@@ -8,8 +8,7 @@
         "fileset.name": "log",
         "http.request.method": "get",
         "http.request.referrer": "http://localhost:5601/app/kibana",
-        "http.response.content_length": 9,
-        "http.response.elapsed_time": 26,
```

Comment:
We should mention the removal of this field in the breaking changes section of the CHANGELOG.

Reply:
Yes, agreed. I've often glossed over the details, for the module-specific fields (e.g. "migrate `apache.access.*`").

But these two deserve a special mention indeed, since they could have been mistaken for ECS fields.